include/ddc/detail/tagged_vector.hpp

```diff
@@ -242,6 +242,8 @@ class TaggedVector : public ConversionOperators<TaggedVector<ElementType, Tags..
     std::array<ElementType, sizeof...(Tags)> m_values;
 
 public:
+    using element_type = ElementType;
```

Comment:
To mimic `std::array` I would expect `value_type`, what do you think ?
```suggestion
    using value_type = ElementType;
```

Reply:
This is fine for me. I used `element_type` to match `Chunk`

Reply:
It seems that there is a `value_type` too. What is the difference?

Reply:
In `ChunkSpan` ? It is inspired from `mdspan`, they seem to define it as `using value_type = remove_cv_t<element_type>`